src/styles/blocks/header.scss

```diff
@@ -0,0 +1,58 @@
+.header {
+  padding: 0 50px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+
+  @include flex(center, space-between);
+}
+
+.logo__link {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  &__list {
+    flex-wrap: wrap;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    @include flex(center, center);
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #060b35;
+    font-weight: 500;
+    font-size: 12px;
+    font-style: normal;
+    line-height: 60px;
+    position: relative;
+
+    @include flex;
+  }
+
+  .is-active,
+  &__link:hover {
+    transition: color 300ms ease;
+    color: $color-links;
```

Comment:
The variable `$color-links` is used here. Make sure it is defined in your SCSS files, as it is not a standard CSS color value.